plugins/basic/python/tk_photoshopcc_basic/__init__.py

```diff
@@ -8,4 +8,5 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-from .plugin_bootstrap import bootstrap_toolkit
\ No newline at end of file
+from .classic_init import toolkit_classic_bootstrap
+from .plugin_init import toolkit_plugin_bootstrap
```

Comment:
So the `bootstrap.py` is called from javascript, so that's a precise operation. But then, what happens next if we didn't use the encapsulation? There would have to be some adding to the python path, some importing etc? For this step, imagine that there were two different ps plugins, both built on top of the adobe tk framework, perhaps tracking different configs? This would not work unless we either kept all the code in a single file or created a module which is scoped by the plugin id `tk_photoshopcc_basic` to ensure uniqueness. Python zen - "Explicit is better than implicit." - clear interfaces and strong encapsulation is a protection mechanism here that allows us to have greater control over future changes. Happy to convinced otherwise, but would love to hear the use case around having another plugin active in PS built on top of the toolkit platform. :)